platform/src/main/java/org/stellar/anchor/platform/config/PropertySep10Config.java

```diff
@@ -15,4 +15,5 @@ public class PropertySep10Config implements Sep10Config {
   private List<String> clientAttributionDenyList;
   private List<String> clientAttributionAllowList;
   private List<String> omnibusAccountList;
+  private boolean requireKnownOmnibusAccount = true;
```

Comment:
Can you add validation for this new parameter here at `PropertySep10Config`? If requireKnownOmnibusAccount is true and omnibusAccountList is empty, we should throw an error to stop the boot because that configuration would prevent any SEP-10 authentication from passing.

We're already doing this kind of validation in [PropertyIntegrationAuthConfig.java#L12](https://github.com/stellar/java-stellar-anchor-sdk/blob/ff95819513c0d381dc3322709ab5e4abcc316790/platform/src/main/java/org/stellar/anchor/platform/config/PropertyIntegrationAuthConfig.java#L12). You basically need to implement the `Validator` interface and its methods.

Reply:
yeah, i'll add the validation. 
as for an empty omnibusAccountList, i think that should be valid, eg: an anchor wants to block access to the Anchor Platform. thoughts on this use case?

Reply:
yup